docs/source/markdown/podman-network.1.md

```diff
@@ -41,10 +41,12 @@ Podman requires specific default IPs and, thus, network subnets.  The default va
 ### Podman network
 The default bridge network (called `podman`) uses 10.88.0.0/16 as a subnet. When Podman runs as root, the `podman` network is used as default.  It is the same as adding the option `--network bridge` or `--network podman`. This subnet can be changed in **[containers.conf(5)](https://github.com/containers/common/blob/main/docs/containers.conf.5.md)** under the [network] section. Set the `default_subnet` to any subnet that is free in the environment. The name of the default network can also be changed from `podman` to another name using the default network key. Note that this is only done when no containers are running.
 
-### Slirp4netns
```

Comment:
Do not remove the slirp4netns section

Reply:
Maybe put the Pasta section above it, though, given that is now the default for rootless.